samples/Nancy.Demo.Async/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "version": "2.0.0-*",
+
+  "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",

```

Comment:
I think we can use what is there with the version removed. no specifics needed for the demos as the name should highlight what it does?